common/changes/@uifabric/experiments/tile-props_2019-07-01-23-47.json

```
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@uifabric/experiments",
+      "comment": "Filter out disallowed props from Tile root element",
+      "type": "hotfix"
```

```suggestion
      "type": "patch"
```

Nit but I'd stick with `patch`, not sure our tooling handles `hotfix` well enough.